src/ovirtapi/transform.js

```diff
@@ -472,9 +472,21 @@ const Cluster = {
 //
 const Nic = {
   toInternal ({ nic }: { nic: ApiNicType }): NicType {
+    const ips = (
+      nic.reported_devices &&
+      nic.reported_devices.reported_device &&
```

Comment:
In this case, better use ternary operator `?:`

Reply:
Ok.  Using a ternary looks like it would be more consistent to the rest of the file.